guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc

```diff
@@ -1,6 +1,10 @@
 [[sect-Administering-Installing_the_Katello_Root_CA_Certificate]]
 === Installing the Katello Root CA Certificate
 
+ifndef::satellite[]
+include::snip_katello-only.adoc[]
+endif::[]
```

Comment:
The reason this section is Katello-only is that `https://_{foreman-example-com}_/pub` does not exist on non-Katello.

The manual import that is below actually does work for non-Katello but you need to modify the path a bit. I think what you can do is get the file that's mentioned in `/etc/httpd/conf.d/05-foreman-ssl.conf` (`apache2` instead of `httpd` on Debian) as `SSLCertificateChainFile` and import that.

If anything, you can say `/etc/puppetlabs/puppet/ssl/certs/ca.pem` for non-Katello (which is the default) can be used.

There is actually a PR that was recently merged (https://github.com/theforeman/foreman/commit/0d08d7a6ffeb267e4c38755bb68a48ae3f57da01) which at least introduces a setting. With that we can define an endpoint to introduce common functionality.

Reply:
Can we break up things that Mel cannot do into separate issues so engineers can do these changes? This smell like something that is a bit more complex. Then we could merge this PR which contains bunch of meaningful changes and improve the guide.